### ISSUE TITLE
No such column: j.fixed_key

### ISSUE THREAD

**Issue by rkoster** [starkandwayne/shield-boshrelease#106]
When using the shield ui I see the following error in `/var/vcap/sys/log/shield/shieldd.log`:
```
2018-01-04 11:41:58.643334246 +000 UTC /var/vcap/packages/shield/bin/shieldd: ERROR: GET /v2/tenants/0ba8c0b6-3976-44eb-b8f5-10a36a82c04e/health errored: no such column: j.fixed_key
```
Each click in the ui generates the above log line. I'm using [v8.3.0](https://github.com/starkandwayne/shield-boshrelease/releases/tag/v8.0.3). And have just one agent and an empty database.

**Comment by jhunt** [comment]
DId you upgrade from 8.0.2 to 8.0.3 (assuming you don't have a future copy of 8.3.0)?

**Comment by rkoster** [comment]
Yes yesterday I started with 8.0.2, and today in the same environment deployed 8.0.3.

**Comment by jhunt** [comment]
Try redeploying 8.0.3 from scratch.

**Comment by jhunt** [comment]
Cool.

Sorry for the inconvenience, we're playing a little fast and loose with the 8.x line right now, but eventually it will stabilized into a real release line.

**Comment by rkoster** [comment]
No problem, thanks for the troubleshooting
